Content.Goobstation.Shared/MartialArts/Events/KravMagaActionEvent.cs

```diff
@@ -15,4 +17,20 @@ namespace Content.Goobstation.Shared.MartialArts.Events;
 /// </summary>
 public sealed partial class KravMagaActionEvent : InstantActionEvent
 {
-}
\ No newline at end of file
+};
+
+/// <summary>
+/// This is for inflicting lung damage
+/// </summary>
+[Serializable, NetSerializable]
+public sealed partial class KravMagaLungPunchEvent : EntityEventArgs
+{
+    public EntityUid Target;
```

Comment:
```suggestion
    [DataField("target")]
    public NetEntity Target { get; }

    [DataField("lungDamage")]
    public int LungDamage { get; }

    public KravMagaLungPunchEvent(NetEntity target, int lungDamage)
    {
        Target = target;
        LungDamage = lungDamage;
```

Reply:
Made it into a Network Entity (not sure if its the best approach but it deals with your serialization)